helmfile.d/upstream/open-policy-agent-gatekeeper/gatekeeper/values.yaml

```diff
@@ -1,44 +1,63 @@
 replicas: 3
+revisionHistoryLimit: 10
 auditInterval: 60
 metricsBackends: ["prometheus"]
 auditMatchKindOnly: false
 constraintViolationsLimit: 20
 auditFromCache: false
 disableMutation: false
 disableValidatingWebhook: false
+validatingWebhookName: gatekeeper-validating-webhook-configuration
 validatingWebhookTimeoutSeconds: 3
 validatingWebhookFailurePolicy: Ignore
 validatingWebhookAnnotations: {}
 validatingWebhookExemptNamespacesLabels: {}
 validatingWebhookObjectSelector: {}
 validatingWebhookCheckIgnoreFailurePolicy: Fail
 validatingWebhookCustomRules: {}
+validatingWebhookURL: null
 enableDeleteOperations: false
 enableExternalData: true
-enableGeneratorResourceExpansion: false
+enableGeneratorResourceExpansion: true
```

Comment:
Questions, do we want to keep override it to be `false`?
As I understand it this will enable one to create `ExpansionTemplate`s which we aren't using atm.
* https://github.com/open-policy-agent/gatekeeper/blob/master/website/docs/expansion.md

Reply:
To me it feels like we should disable them by default until we decide to use them

Reply:
After rereading that page again, I would also say that we could disable it. I thought first it was needed for gatekeeper to do some validation steps. But it seems to be a way to tell gatekeeper to create a dummy object (pod) for a resource to be validated.